build.cake

```diff
@@ -91,6 +91,9 @@ Task("Publish")
 });
 
 Task("Default")
-	.IsDependentOn("Publish");
+	.IsDependentOn("Post-Build");
```

Comment:
@agc93 in this case, it is not "strictly" required, however, it may come to this if/when we actually do a publish to Gallary, so we might want to add some criteria to only do the publish when we are in the correct state.  i.e. not a PR, running on the actual master cake repository, and runnign on appveyor.  This would be similar to the criteria you can see on the publish steps on the main cake repo.